api/types/plugin.go

```diff
@@ -76,6 +78,8 @@ const (
 	PluginSubkindMDM = "mdm"
 	// PluginSubkindAccess represents access request plugins collectively
 	PluginSubkindAccess = "access"
+	// PluginSubkindAccessGraph represents access graph plugins collectively
+	PluginSubkindAccessGraph = "accessgraph"
```

Comment:
How is this relevant to this PR?

Reply:
This works as a plugin subtype to easily identify which plugins are associated with access graph. It's used by the enterprise PR